.github/workflows/build-ci.yml

```diff
@@ -53,10 +94,9 @@ jobs:
     - name: Install Python dependencies
       if: ${{ matrix.python-version != 'none' }}
       run: pip install autograd h5py jax jaxlib matplotlib mpi4py numpy parameterized pytest scipy
-    - name: Autoreconf
-      run: autoreconf -if
     - name: configure
-      run: ./configure --with-hdf5 --without-scheme --enable-shared ${{ matrix.configure-options }}
+      run: |
+        sh autogen.sh --without-scheme --enable-shared CC=${MY_CC} LDFLAGS="${MY_LDFLAGS}" CPPFLAGS="${MY_CPPFLAGS}" ${{ matrix.configure-options }}
```

Comment:
Shouldn't this be setting `CXX=${MY_CXX}` to build Meep?   We don't really use the C compiler (`CC`).

Reply:
Fixed.